orderbook/src/database/orders.rs

```diff
@@ -248,7 +250,16 @@ impl OrderStoring for Postgres {
                 WHEN 'sell' THEN sum_sell < sell_amount \
                 WHEN 'buy' THEN sum_buy < buy_amount \
             END) AND \
-            ($7 OR NOT invalidated);";
+            ($7 OR NOT invalidated) \
+        ORDER BY unfiltered.creation_timestamp \
+        LIMIT $8 \
+        OFFSET $9;";
```

Comment:
If i don't remember bad, these formatting syntax was very powerful, and would allow u to use "named placeholders" instead of numbers. I think its the same efficiency and more readable when u have so many params --> hard to spot a mistake.

Reply:
Is the OFFSET the same as the `current_page`? 

afaik u can implement the navigation using an offset, or a a page number but is not the same. 
U would need to multiply the `page_size X (page_number -1)` to get the offset

i.e. 
page 1, 100 ==> offset 0
page 2, 100 ==> offset 100